src/components/ReportWelcomeText.tsx

```diff
@@ -123,12 +124,13 @@ function ReportWelcomeText({report, policy}: ReportWelcomeTextProps) {
                     (welcomeMessage?.messageHtml ? (
                         <PressableWithoutFeedback
                             onPress={() => {
-                                if (!canEditPolicyDescription) {
+                                if (!canEdit) {
```

Comment:
Can you change it to more descriptive name like canEditReportDescription

Also update similar usage in line 159